static/app/components/feedback/useFeedbackListQueryKey.tsx

```diff
@@ -38,38 +44,47 @@ export default function useFeedbackListQueryKey({organization}: Props): ApiQuery
     // the user wants to see fresher content (like, after the page has been open
     // for a while) they can trigger that specifically.
 
+    // The issues endpoint cannot handle when statsPeroid has a value of "", so
+    // we remove that from the rest and do not use it to query.
     const {statsPeriod, ...rest} = queryView;
-    const now = Date.now();
-    const statsPeriodMs = intervalToMilliseconds(statsPeriod);
-    return statsPeriod
-      ? {
-          ...rest,
-          start: new Date(now - statsPeriodMs).toISOString(),
-          end: new Date(now).toISOString(),
-        }
-      : // The issues endpoint cannot handle when statsPeroid has a value of "", so
-        // we remove that from the rest and do not use it to query.
-        rest;
-  }, [queryView]);
+
+    if (prefetch) {
```

Comment:
what is the `prefetch` doing? / why do we need it?

Reply:
added a comment about it; It's not the best name

Reply:
taking suggestions though!

Reply:
so `prefetch` refers to fetching into the future? maybe we could rename it to `futureFetch` or something similar?